lib/build.gradle

```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 group 'com.dynatrace.metric.util'
-version = '0.1.1'
+version = '0.1.2'
```

Comment:
Since we are now throwing in a case where we previously didn't throw, should we bump to `0.2.0`? /cc @arminru

Reply:
https://semver.org/#spec-item-4

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Either is fine but yeah let's do `0.2.0`.

Reply:
Even though none of the APIs changed? Only the behavior is different now, but yeah, I guess it makes sense to bump the minor version here because that could be unexpected.